src/containerapp/azext_containerapp/_client_factory.py

```diff
@@ -147,3 +147,10 @@ def k8s_extension_client_factory(cli_ctx, subscription_id=None):
 
     r = get_mgmt_service_client(cli_ctx, SourceControlConfigurationClient, subscription_id=subscription_id)
     return r.extensions
+
+
+def connected_k8s_client_factory(cli_ctx, subscription_id=None):
+    from azext_containerapp.vendored_sdks.hybridkubernetes import ConnectedKubernetesClient
```

Comment:
The ConnectedKubernetesClient need to import pkg `'azure-mgmt-hybridkubernetes==1.1.0'`, according to previous communication, it is recommended to use vendored sdk.

Reply:
There are 2 options to choose from after entering core:
If the code depend on is not much, we can consider continuing to vendor the small amount of code we need.
If the code depend on is a lot, and we want to update the code with the package in the future, we can consider adding the dependency of `azure-mgmt-hybridkubernetes==1.1.0` in Core.